### PR TITLE
Deprecate tctl in favor of new Temporal CLI

### PR DESCRIPTION
**Pull request by longquanzheng**
for https://github.com/indeedeng/iwf/issues/313

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -77,50 +77,52 @@ Then run the whole integ test suite against Cadence+Temporal service by this com
 
 Or run `make temporalIntegTests` if you don't want to wait :/
 
-### Option 2: Run with your own Temporal service
+To run the server, if you are in an IDE, you can run the main function in `./cmd/main.go` with argument `start`.
 
-NOTE: For local testing it's recommended to use [temporal docker-compose](https://github.com/temporalio/docker-compose)
+Alternatively, you could just build the binary by `make bins` and then run `./iwf-server start`
```
Comment (zklgame) [inline CONTRIBUTING.md:82]: Or, directly using the command line in a terminal under the iwf folder:
```
 go run cmd/server/main.go start
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -77,50 +77,52 @@ Then run the whole integ test suite against Cadence+Temporal service by this com
...
+   attributes:`temporal operator search-attribute list`
 
-Then run the whole integ test suite against Cadence+Temporal service by this command:
+Then run the whole integ test suite against Temporal service by this command:
 
 `make temporalIntegTests`
 
+To run the server, if you are in an IDE, you can run the main function in `./cmd/main.go` with argument `start`.
+
+Alternatively, you could just build the binary by `make bins` and then run `./iwf-server start`
```
Comment (zklgame) [inline CONTRIBUTING.md:124]: These two sentences here are the same as the line 80 and 82?

**Review by zklgame — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (longquanzheng) [reply CONTRIBUTING.md:82]: good idea. I never used this but just tried it and it works :D

File: CONTRIBUTING.md (hunk 2 above)
Comment (longquanzheng) [reply CONTRIBUTING.md]: yeah that's not nice. I just moved them to the same place.

**Pull request merged**